lib/file_scope.rb

```diff
@@ -16,6 +16,13 @@
 # you may find current contact information at www.suse.com
 
 class FileScope < Machinery::Object
+  def write_files_tarball(destination)
+    FileUtils.cp(
+      File.join(scope_file_store.path, "files.tgz"),
+      destination
+    )
+  end
```

Comment:
Doesn't this method belong to the `SystemFileUtils` class? I think we should have only one class which is responsible for the structure of how extracted files are stored on disk as part of the system description.